pom.xml

```diff
@@ -50,8 +52,56 @@
         <dependency>
             <groupId>com.auth0</groupId>
             <artifactId>java-jwt</artifactId>
-            <version>3.15.0</version>
+            <version>3.18.1</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.jetbrains/annotations -->
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <version>22.0.0</version>
         </dependency>
```

Comment:
Could you explain why this is needed for this change please?

Reply:
You import the `NotNull` annotation from Jetbrains in the 'JWTAccessManager` so I added it as a dependency, but I've done some research and realized that it isn't really a Runtime dependency. Those annotations are only used by tools that do a static code analysis. Following the instructions from Jetbrains on there README it should be enough to add it as with a `provided` scope.
https://github.com/JetBrains/java-annotations#using-the-annotations
The tools that run and analyze the code should provide the package. I will add the scope to the dependency to make it no longer required.

Reply:
Ok I see, it's been awhile I don't even remember that. Feel free to remove the dependency and its usage, I don't think it brings much value here anyway, or just change the scope as you mentioned.

Reply:
I will include it with the scope because removing them will make IntelliJ complain about them missing -_-

Reply:
I see.. they're used by Javalin's access manager and IntelliJ doesn't like it when implementations don't use the same annotations. That should be fine then